pipeline/scripts/generate_truth_data.jl

```diff
@@ -0,0 +1,49 @@
+using DrWatson
+@quickactivate "Analysis pipeline"
```

Comment:
Sad branching problems I see! Note to ourselves for the future is a branch is based on another branch we can't really use squash merging (or we can but its a bit more painful)